NGitLab/IProjectBadgeClient.cs

```diff
@@ -26,24 +26,32 @@ public interface IProjectVariableClient
 {
     IEnumerable<Variable> All { get; }
 
-    Variable this[string key] { get; }
+    Variable this[string key, string environmentScope = null] { get; }
```

Comment:
```suggestion
    Variable this[string key] { get; }
    
    Variable this[string key, string environmentScope] { get; }
```

I would separated instead of using optional param, as adding it (even if optional) is a change of the signature, thus, a breaking change.